build.gradle

```diff
@@ -47,10 +47,13 @@ dependencies {
         exclude(group: 'org.openjfx')
     }
     implementation("io.github.kostaskougios:cloning:1.10.3")
-
-    testImplementation("org.junit.jupiter:junit-jupiter-api:${junitVersion}")
-    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:${junitVersion}")
+    implementation 'org.openjfx:javafx-controls:11'
+    implementation 'org.openjfx:javafx-fxml:11'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
```

Comment:
Why is this hardcoded to 5.7.0 when we have `${junitVersion}`

Reply:
Needed to run TestFx.

Reply:
But why do you change it here to be hardcoded

Reply:
It wasn't previosuly hardcoded and you hsouldn't change the test dependencies, if you want to change a version just change the variable

Reply:
It wasn't working otherwise, or else we would have done it elsewhere.

Reply:
Also none of these follow the modern gradle build format: `implementation("")`

Reply:
@alvinayangg Can you answer this? I don't know what's going on in `build.gradle`.

Reply:
> It wasn't working otherwise, or else we would have done it elsewhere.

Sorry, what did you try, what wasn't working, what is the error?

Ref: https://stackoverflow.com/help/how-to-ask

Reply:
Alvina will respond to this.

Reply:
I feel like hardcoding isn't the best approach to this. If there was an issue with using the `${junitVersion}` variable, we need to look into that further. @snkrs23, could you please provide more details about the error you encountered? Also, maybe try rewriting to the proper gradle format and try using this and let me know if there's still issues.

Reply:
It would just show that TestFx isn't installed, I don't have any idea how you would fix that and given that it doesn't affect any SOLID or CA principles, I'd just leave it alone.

Reply:
Thanks for ficing it @alvina-yang , thats exactly what I was saying